Extensions/bezier/bezier.lisp

```diff
@@ -209,6 +209,8 @@
 			(transform-segment transformation segment))
 	       (%segments path))))
 
+;;; FIXME! This overrides region-equal in Core/clim-basic/regions.lisp
+;;; -- we probably don't want that!
 (defmethod region-equal ((p1 point) (p2 point))
```

Comment:
I wonder, if we should use commented code from `Core/clim-basic/coordinates.lisp`, which does the same thing (that is - it works with epsilon instead of exact coordinates), what makes "default" `region-equal` behave exactly like this method. What do you think?